src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/AttendancesResponse.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.probation.courtcaseservice.controller.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@ApiModel(description = "Attendance Wrapper")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(Include.NON_NULL)
+public class AttendancesResponse {
```

Comment:
This probably ought to be called ConvictionsResponse eventually as we'll be adding more data in but can leave it as it is for now as it does only have attendance data.

Reply:
In terms of the naming - the tickets were all written around "attendance". The sample array had "attendances" We changed the parameter being passed because that made sense to the caller but not so sure about the renaming of everything